cypress/integration/SelectLabs.spec.ts

```diff
@@ -304,7 +304,7 @@ describe('Select', () => {
 
         context('when the space key is pressed', () => {
           beforeEach(() => {
-            cy.findByLabelText('Label').type(' ');
+            cy.findByLabelText('Label').type(' ', {force: true}); // disable event.preventDefault checks
```

Comment:
Not 100% sure about this. The keyboard plugin does allow disabling of `defaultPrevented` checks if `force: true` is passed. This technically bypasses other checks as well like visibility or interactability, but those checks are covered in other tests. This is required because the Select uses a `<button>` element, but doesn't handle the space bar the same way a button does. A button triggers click on keyup of a spacebar, which a select triggers a click on the keydown of a spacebar key event.